test/verify/check-loopback

```diff
@@ -35,7 +35,13 @@ class TestLoopback(MachineCase):
         else:
             cockpit_ws = "/usr/libexec/cockpit-ws"
 
+        # Make sure SSH is started and sane
+        m.wait_execute()
```

Comment:
Doesn't `m.spawn` indirectly do this already? From what I can see, it calls `_ensure_ssh_master`.